commodity.py

```diff
@@ -39,27 +44,30 @@ def export(data, kind=COMMODITY_DEFAULT, filename=None):
 
         rowheader = sep.join((data['lastSystem']['name'], data['lastStarport']['name']))
 
-    h = open(filename, 'wt')  # codecs can't automatically handle line endings, so encode manually where required
-    h.write(header)
-
-    for commodity in data['lastStarport']['commodities']:
-        line = sep.join((
-            rowheader,
-            commodity['name'],
-            commodity['sellPrice'] and str(int(commodity['sellPrice'])) or '',
-            commodity['buyPrice'] and str(int(commodity['buyPrice'])) or '',
-            str(int(commodity['demand'])) if commodity['demandBracket'] else '',
-            bracketmap[commodity['demandBracket']],
-            str(int(commodity['stock'])) if commodity['stockBracket'] else '',
-            bracketmap[commodity['stockBracket']]
-        ))
-
-        if kind == COMMODITY_DEFAULT:
-            line = sep.join((line, str(int(commodity['meanPrice'])), str(commodity['id']), data['timestamp'] + '\n'))
-
-        else:
-            line = sep.join((line, data['timestamp'] + '\n'))
-
-        h.write(line)
-
-    h.close()
+    with open(filename, 'wt') as h:  # codecs can't automatically handle line endings, so encode manually where required
+        h.write(header)
+
+        for commodity in data['lastStarport']['commodities']:
+            line = sep.join((
+                rowheader,
+                commodity['name'],
+                commodity['sellPrice'] and str(int(commodity['sellPrice'])) or '',
+                commodity['buyPrice'] and str(int(commodity['buyPrice'])) or '',
+                str(int(commodity['demand'])) if commodity['demandBracket'] else '',
+                bracketmap[commodity['demandBracket']],
+                str(int(commodity['stock'])) if commodity['stockBracket'] else '',
+                bracketmap[commodity['stockBracket']]
+            ))
+
+            if kind == COMMODITY_DEFAULT:
+                line = sep.join((
+                    line,
+                    str(int(commodity['meanPrice'])),
+                    str(commodity['id']),
+                    data['timestamp'] + '\n')
+                )
```

Comment:
```suggestion
                line = sep.join(
                    (
                        line,
                        str(int(commodity['meanPrice'])
                    ),
                    str(commodity['id']),
                    data['timestamp'] + '\n')
                )
```
Or similar, to better show that the tuple doesnt contain everything here. Though Im not sure if this is a bug or not.

Reply:
It's actually:

```python
                line = sep.join(
                    (
                        line,
                        str(int(commodity['meanPrice'])),
                        str(commodity['id']),
                        data['timestamp'] + '\n'
                    )
                )
```
I'll move that 2nd-last `)` down a line to make it more obvious

Reply:
Actually, I'll just split it as above comment.